### ISSUE TITLE
Installing

### ISSUE THREAD

**Issue by jotadominguez** [mjg59/python-decora#5]
Could you please give more information on how to install it and use it.
Thanks

**Comment by KShivum** [comment]
I don't think it works anymore, I am not able to run any of the files. Sucks that home assistant links this repo as what you should use to get the api key
